packages/core/src/storages/utils.ts

```diff
@@ -3,7 +3,17 @@ import type { Dictionary, StorageClient } from '@crawlee/types';
 import { KeyValueStore } from './key_value_store';
 import { Configuration } from '../configuration';
 
+/**
+ * Options for purging default storage.
+ */
 interface PurgeDefaultStorageOptions {
+    /**
+     * If set to `true`, calling multiple times will only have effect at the first time.
+     *
+     * @type {boolean}
+     * @optional
+     * @default false
```

Comment:
```suggestion
```
this only duplicates the information we already have on type level (i could understand the `@default` but since the default is false, i dont think we need to specify it)